fastlane/Fastfile

```diff
@@ -350,26 +350,27 @@ end
         method: "enterprise",
         provisioningProfiles: {
           "com.codality.NotationalFlow.Internal" => "match InHouse com.codality.NotationalFlow.Internal",
-          "com.codality.NotationalFlow.Internal.Share" => "match InHouse com.codality.NotationalFlow.Internal.Share"
+          "com.codality.NotationalFlow.Internal.Share" => "match InHouse com.codality.NotationalFlow.Internal.Share",
+          "#{APP_STORE_BUNDLE_IDENTIFIER}.Internal.Intents" => "match InHouse #{APP_STORE_BUNDLE_IDENTIFIER}.Internal.Intents"
```

Comment:
The code here could have been updated a bit more, like I did for the calls to `match`, but I decided not to go down that way and leave the diff smaller.

Reply:
Given that `APP_STORE_BUNDLE_IDENTIFIER` is equal to `com.codality.NotationalFlow`, wouldn't it make sense to use `#{APP_STORE_BUNDLE_IDENTIFIER}.Internal` in line 353 too, instead of repeating `com.codality.NotationalFlow.Internal` as a literal?

Reply:
Also, I'd love to see more DRY here tho, like having a `simplenote_provisioning_profiles` method to provide those.

My attempt at this:
```ruby
# in_house should be `nil` for AppStore, and `"Internal"` or `"Alpha"` for InHouse builds
def simplenote_provisioning_profiles(in_house_suffix = nil)
  root_bundle_id = [APP_STORE_BUNDLE_IDENTIFIER, in_house_suffix].compact.join('.')
  match_type = in_house_suffix.nil? ? "AppStore" : "InHouse"
  bundle_ids = [
    root_bundle_id,
    "#{root_bundle_id}.Share",
    "#{root_bundle_id}.Intents"
  ]
  bundle_ids.map { |key| [key, "match #{match_type} #{key}"] }.to_h
end
```

* Usage here: `provisioningProfiles: simplenote_provisioning_profiles('Internal')`
* Usage on line 402: `provisioningProfiles: simplenote_proviosining_profiles`

---

Alternatively, if you prefer, you could also make the method take the whole `root_bundle_id` as a parameter of the method, maybe making `APP_STORE_BUNDLE_IDENTIFIER` the default value for that parameter, and compare the param against that `APP_STORE_BUNDLE_IDENTIFIER` to determine the `match_type`. Up to you.

Reply:
> wouldn't it make sense to use... Also, I'd love to see more DRY here tho...

Yeah, totally! I didn't go down that path because I felt like I had already made too many changes in this PR, which was about adding support for Intents. I probably should have done the refactoring step in isolation first, then add the support on top of it.

I've already spent time to write this commend and DRY the implementation, though, so I might as well keep going and implement your suggestion 😄